templates/Signify/Factory/Models/TableBlock.ss

```diff
@@ -3,7 +3,13 @@
     <h2>$Title</h2>
   <% end_if %>
   <% if $TableDescription %>
-    $TableDescription
+    <div class="table-block__description">
```

Comment:
This is a change to formatting which **could affect other sites**.

It would be better if you copied this template into your theme and changed it there.